doc/rst/source/datasets/remote-data.rst

```diff
@@ -0,0 +1,134 @@
+Remote Data Sets
+================
+
+A *remote data set* is a data set that is stored on one or more remote servers. It may
+be a single grid file or a collection of subset tiles making up a larger grid. They
+are not distributed with GMT or installed during the installation procedures.
+GMT offers several remote global data grids that you can access via our *remote file* mechanism.
+The first time you access one of these files, GMT will download the file (or a subset tile) from
+the selected GMT server and save it to the *server* directory under your GMT user directory [~/.gmt].
+From then on we read the local file from there.
+
+By using the remote file mechanism you should know that these files, on the server, will change
+from time to time (i.e., new versions are released, a problem in one file is fixed, or a dataset
+becomes obsolete), and GMT will take actions accordingly.  It is our policy to only supply the *latest*
+version of any dataset that undergoes revisions.  If you require previous versions for your work you
+will need to get those data from the data provider separately.  Unless you deactivate the remote data service,
+GMT will do the following when you request a remote file in a GMT command:
+
+#. We check if the locally cached catalog with information about the data available from the server
+   is up-to-date or if it needs to be refreshed.  If the file is older that the :term:`GMT_DATA_SERVER_UPDATE`
+   limit then we refresh the catalog.
+#. When the catalog is refreshed, we determine the publication date for each dataset on the server,
+   and if any local copies you may have are now obsolete we will remove them to force a re-download from the server.
```

Comment:
It would be useful if we can list all the available dataset here, and then link to the section titles below.

Reply:
You mean just a very short list of the types, like Earth DEM [link], Images [link], crustal age [link], etc?

Reply:
This is what I have done:
![image](https://user-images.githubusercontent.com/3974108/87212277-b0f26480-c2eb-11ea-9331-8803478772eb.png)

Reply:
OK, that is great, can you commit it? I just pushed the GMT_DATA_UPDATE_INTERVAL change

Reply:
Done.